.github/workflows/test.yml

```diff
@@ -13,27 +13,23 @@
 # Requires scripts:
 # - bin/test
 
-name: <track> / Test
+name: COBOL / Test (Linux)
 
-on:
-  push:
-    branches: [main]
-  pull_request:
-  workflow_dispatch:
+on: [push]
```

Comment:
Why did you change the original settings?

Reply:
Just for test purposes.

Reply:
I need to move the CI pipeline out of this PR since it is getting unnecessarily complex otherwise. 
But I'll keep that in mind!